### DISCUSSION TITLE
remove helmrelease without removing actual helm chart

### DISCUSSION THREAD

**Discussion by mo-saeed** [fluxcd/flux2#2790] [General] [answered]
Hi,

Is there anyway to delete helmrelease without performing actual `helm uninstall` for the chart ? 

Thanks in advance

**Comment by stefanprodan** [comment, marked as answer]
Yes, you can do `flux suspend helmrelease <name>`, then delete the HelmRelease from Git. Or you can set `spec.suspend` to `true` in Git, commit & push & sync, then delete it from Git and push.
